pkg/results/report.go

```diff
@@ -0,0 +1,144 @@
+package results
+
+import (
+	"bytes"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/golang/glog"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+// Options holds the configuration options for connecting to the remote aggregation server
+type Options struct {
+	address  string
+	certFile string
+	keyFile  string
+	caFile   string
+}
+
+// Bind adds flags for the options
+func (o *Options) Bind(flag *flag.FlagSet) {
+	flag.StringVar(&o.address, "report-address", "", "Address of the aggregate reporting server.")
```

Comment:
Why? We can just use a specific log line or even junit and find it via ci-search?

Reply:
If we are building an aggregation service I'd like to see some design first

Reply:
Design has been hashed out pretty well at this stage.

Reply:
Where?